pyzscaler/utils.py

```diff
@@ -44,12 +44,12 @@ class Iterator(APIIterator):
 
     page_size = 100
 
-    def __init__(self, api, path="", params={}, **kw):
+    def __init__(self, api, path: str = "", params: dict = None, **kw):
```

Comment:
I saw that pylint complained about using {} as default. So I changed it to None instead even though the value will not get muteted.